lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,19 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(locales_path = GettextSetup.locales_path)
+      mergepot = GettextSetup.config['project_name'] + '.pot'
+      cmd = "msgcat #{locales_path}/*.pot > #{locales_path}/#{mergepot}"
```

Comment:
The intention here was to replace the existing one regardless, but I can output a message warning users of the override.